add_to_app/books/ios_books/IosBooks/SceneDelegate.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
```

Comment:
Is this one of your files or an auto-generated one? If it's the former, add a copyright header.

Reply:
done